tests/config.neon

```diff
@@ -1,13 +1,19 @@
-application:
-	errorPresenter: Error
-	mapping:
-		*: App\*Module\Presenters\*Presenter
+parameters:
+	math:
+		functions:
+			- sin
+			- cos
+			- tan
+			- cotan
+			- log
+			- log\d*
+			- ln
+			- sqrt
 
 services:
 	- Mathematicator\Engine\QueryNormalizer
 	- Mathematicator\Numbers\NumberFactory
 	- Mathematicator\NumberRewriter
-	- Mathematicator\Tokenizer\Tokenizer
-	- Mathematicator\Tokenizer\TokensToLatex
+	- Mathematicator\Tokenizer\Tokenizer(%math%)
+	- Mathematicator\Tokenizer\TokensToLatex(%math.functions%)
```

Comment:
this section is deprecated in related package.

Reply:
So how to solve it?

Reply:
@janbarasek It is deprecated in dev stability version of the tokenizer package, so I propose to change this test when the tokenizer will be stable, otherwise it will fail.

Reply:
I think it will be better if I release a stable version of the second package first and this PR will wait. Otherwise, we would have to resolve this error again.

Reply:
Now it's fixed.